lib/client.js

```diff
@@ -0,0 +1,98 @@
+"use strict";
+
+const Promise = require("bluebird");
```

Comment:
http://programmers.stackexchange.com/questions/278778/why-are-native-es6-promises-slower-and-more-memory-intensive-than-bluebird